benchmarks/ig-en/jw300-baseline/README.md

```diff
@@ -0,0 +1,62 @@
+# IGBO TO ENGLISH
+**Authors:**
+Dino Anastasopoulos (1900661)
```

Comment:
Please remove your student numbers unless you want the whole world to know them ;)